cli/lib/kontena/cli/services/scale_command.rb

```diff
@@ -13,7 +13,7 @@ def execute
       token = require_token
       spinner "Scaling #{pastel.cyan(name)} to #{instances} instances " do
         deployment = scale_service(token, name, instances)
-        wait_for_deploy_to_finish(token, deployment)
+        wait_for_deploy_to_finish(deployment)
```

Comment:
There could be a `--wait` here?

Reply:
Should probably be a separate PR for fixing `kontena service scale`, it's not that relevant for this PR:

* Add the `kontena service scale --no-wait` flag
* Fix the `GridServices::Scale` mutation to set the `GridService.state` re #2290

Reply:
#3298